python/manual-instrumentation/server.py

```diff
@@ -0,0 +1,70 @@
+from time import sleep
+
+from flask import Flask
+from opentelemetry import trace
+from opentelemetry.sdk.resources import Resource
+from opentelemetry.instrumentation.flask import FlaskInstrumentor
+from opentelemetry.exporter.otlp.proto.grpc.trace_exporter import (
+    OTLPSpanExporter,
+)
+from opentelemetry.sdk.trace import TracerProvider
+from opentelemetry.sdk.trace.export import BatchSpanProcessor
+from opentelemetry.trace.status import Status, StatusCode
+
+app = Flask(__name__)
+
+# activate flask-instrumentation during its initialization
+FlaskInstrumentor().instrument_app(app)
+
+# resource can be required for some backends like wavefront
+# note: change the value of service.name attribute to your service name
+resource = Resource(attributes={
+    "service.name": "myPythonService"
+})
+
+span_exporter = OTLPSpanExporter(
+    # optional, these are default values
+    # endpoint="myCollectorURL:4317",
```

Comment:
Is the default for `endpoint` `localhost:4317`? If so it would be good to update the value here.

Reply:
Yup, the default value is 'localhost:4317'. I'll update it.